packages/owl-bot/src/copy-code.ts

```diff
@@ -272,7 +272,7 @@ export async function copyCodeAndCreatePullRequest(
   const token = await octokitFactory.getGitHubShortLivedAccessToken();
   // Octokit token may have expired; refresh it.
   const octokit = await octokitFactory.getShortLivedOctokit(token);
-  if (await copyExists(octokit, destRepo, dest.sourceCommitHash)) {
+  if (await copyExists(octokit, destRepo, dest.sourceCommitHash, 100)) {
```

Comment:
Should this be 1000?

Reply:
This is the final check before creating a pull request, to make sure a parallel process didn't create the same pull request.  That would be very recent because the processes are running in parallel, so checking the most recent 100 is sufficient.  Also, in this case we deliberately omit a cache.